intercom_flutter/lib/intercom_flutter.dart

```diff
@@ -128,4 +128,22 @@ class Intercom {
   static Future<void> handlePush(Map<String, dynamic> message) async {
     return IntercomFlutterPlatform.instance.handlePush(message);
   }
+
+  /// To display an Article, pass in an [articleId] from your Intercom workspace.
+  ///
+  /// An article must be ‘live’ to be used in this feature.
+  /// If it is in a draft or paused state,
+  /// end-users will see an error if the app tries to open the content.
+  static Future<void> displayArticle(String articleId) async {
```

Comment:
If this can crash if the feature is not available, maybe we should add a comment here too until it's fixed by Intercom